ohara-manager/client/package.json

```
@@ -16,10 +16,10 @@
     "test:watch": "yarn test",
     "test:report": "cross-env CI=true yarn test --reporters=default --reporters=jest-junit",
     "test:coverage": "yarn test --coverage --watchAll=false",
-    "test:api:open": "cypress open --config integrationFolder=cypress/api",
-    "test:api:run": "cypress run --config integrationFolder=cypress/api",
-    "test:e2e:open": "cypress open",
-    "test:e2e:run": "cypress run"
+    "test:api:open": "cross-env CONFIG_FILE=api cypress open",
```

Let's add a prefix for this env, making it more explicit  :)

```suggestion
    "test:api:open": "cross-env CYPRESS_CONFIG_FILE=api cypress open",
```

---

Thanks! Edit right away.